prediction_market_agent/agents/blockchain_coding_agent/functions.py

```diff
@@ -25,47 +28,44 @@ def tavily_search(
     return tavily_search_pmat(query=query)
 
 
-def fetch_read_methods_from_blockscout(contract_address: str) -> t.Any:
-    w3 = OmenConditionalTokenContract().get_web3()
-    if not is_contract(w3, Web3.to_checksum_address(contract_address)):
-        raise ValueError(f"{contract_address=} is not a contract on Gnosis Chain.")
-    read_not_proxy = fetch_read_methods(contract_address)
-    read_proxy = fetch_read_methods_proxy(contract_address)
-    # ToDo - Fetch write methods
-    return read_not_proxy + read_proxy
+def get_rpc_endpoint() -> str:
+    # return "https://rpc.gnosischain.com"
+    return "http://localhost:8545"
```

Comment:
Why did you change this back to a hard-coded string?

Reply:
My mistake. Reverting and closing this.